controller.py

```diff
@@ -43,108 +52,46 @@ def create_deployment_object_from_file(fullpath: str=""):
-#def delete_deployment(api):
-#    # Delete deployment
-#    resp = api.delete_namespaced_deployment(
-#        name=DEPLOYMENT_NAME,
-#        namespace="default",
-#        body=client.V1DeleteOptions(
-#            propagation_policy="Foreground", grace_period_seconds=5
-#        ),
-#    )
-#    print("\n[INFO] deployment `nginx-deployment` deleted.")
-
-def list_pods(api):
-    print("Listing pods with their IPs:")
-    ret = api.list_pod_for_all_namespaces(watch=False)
-    for i in ret.items:
-        print("%s\t%s\t%s" % (i.status.pod_ip, i.metadata.namespace, i.metadata.name))
-
-def scale_deployment_object(deployment, scalingobject, elasticityobject, publishingRate):
-    #print(scalingobject.elasticityCapacity)
-    desiredReplicas, bufferedReplicas = scalingobject.calcReplicas(publishingRate,deployment.spec.replicas)
-    print("Computed desired replicas: ", desiredReplicas, " and buffered: ", bufferedReplicas)
-    #update elasticity spec of caus
-    deployment.spec.replicas = desiredReplicas
-    elasticityobject.elasticityBufferedReplicas = bufferedReplicas
+def scale_deployment(deployment: client.V1Deployment, caus: CAUS, elasticity: Elasticity, publishing_rate: float) -> client.V1Deployment:
+    desired_replicas, buffered_replicas = caus.calculate_replicas(publishing_rate, deployment.spec.replicas)
+    print(f"Computed desired replicas: {desired_replicas} and buffered: {buffered_replicas}")
+    deployment.spec.replicas = desired_replicas
+    elasticity.buffered_replicas = buffered_replicas
     return deployment
 
 def main():
     print("Load kube config...")
     try:
-        config.load_incluster_config()
-    except config.ConfigException:
+        kubernetes_config.load_incluster_config()
+    except kubernetes_config.ConfigException:
         print("didnt find incluster config, loading file manually...")
-        config.load_kube_config(config_file=os.environ.get('KUBECONFIG','k8s-cluster3-admin.conf'))
+        kubernetes_config.load_kube_config(config_file=os.environ.get('KUBECONFIG','k8s-cluster3-admin.conf'))
 
     #setup deployment, prometheus monitoring, scaling method etc
     #initialize necessary apis
     core_api = client.CoreV1Api()
     apis_api = client.AppsV1Api()
-
-    #create and deploy deploymentspecifications
-    deployment = create_deployment_object_from_file()
-    #create_deployment(apis_api, deployment)
-
-    #setup prometheus (monitoring and api interface)
-    myMonitor = prometheusMonitor()
-
-    #setup elasticity; TODO do so from config file?
-    myElasticity = elasticity(elasticityCapacity=8, elasticityMinReplicas=1, elasticityMaxReplicas=10, elasticityBufferThreshold=50.0,elasticityBufferInitial=1, elasticityBufferedReplicas=1)
+    deployment: client.V1Deployment = create_deployment()
+    monitor = PrometheusMonitor()
+    elasticity = Elasticity(
+            capacity          = config.getint('elasticity', 'elastic-capacity',           fallback = 8),
```

Comment:
new programming language, but still not sure if we want to have alignments in the code

Reply:
Ah, right, there was something.

Reply:
Well, `black` certainly fixed that problem…